### ISSUE TITLE
1.1.0-pre2 image is failing

### ISSUE THREAD

**Issue by carsonip** [graphite-project/docker-graphite-statsd#17]
Using the latest image: 1.1.0-pre2

Here's the logs:
```
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/my_init.d/01_conf_init.sh...
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 14
Nov 29 05:26:01 4d5ab037c9ef syslog-ng[36]: syslog-ng starting up; version='3.5.6'
Traceback (most recent call last):
  File "/opt/graphite/bin/carbon-cache.py", line 32, in <module>
    run_twistd_plugin(__file__)
  File "/opt/graphite/lib/carbon/util.py", line 111, in run_twistd_plugin
    config.parseOptions(twistd_options)
  File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 624, in parseOptions
    usage.Options.parseOptions(self, options)
  File "/usr/local/lib/python2.7/dist-packages/twisted/python/usage.py", line 277, in parseOptions
    self.postOptions()
  File "/usr/local/lib/python2.7/dist-packages/twisted/scripts/_twistd_unix.py", line 77, in postOptions
    app.ServerOptions.postOptions(self)
  File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 635, in postOptions
    % (self['logger'], e))
twisted.python.usage.UsageError: Logger 'None' could not be imported: No module named 'None'
Traceback (most recent call last):
  File "/opt/graphite/bin/carbon-aggregator.py", line 32, in <module>
    run_twistd_plugin(__file__)
  File "/opt/graphite/lib/carbon/util.py", line 111, in run_twistd_plugin
    config.parseOptions(twistd_options)
  File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 624, in parseOptions
    usage.Options.parseOptions(self, options)
  File "/usr/local/lib/python2.7/dist-packages/twisted/python/usage.py", line 277, in parseOptions
    self.postOptions()
  File "/usr/local/lib/python2.7/dist-packages/twisted/scripts/_twistd_unix.py", line 77, in postOptions
    app.ServerOptions.postOptions(self)
  File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 635, in postOptions
    % (self['logger'], e))
twisted.python.usage.UsageError: Logger 'None' could not be imported: No module named 'None'
...
```

**Comment by deniszh** [comment]
Oh, my bad, fixing

**Comment by deniszh** [comment]
Fixed in 1.1.0-pre3, 1.1.0-pre2 release deleted, pushed graphiteapp/graphite-statsd:1.1.0-pre3 and graphiteapp/graphite-statsd:latest manually, graphiteapp/docker-graphite-statsd:latest in in autobuild queue.
